frontend/src/concepts/pipelines/context/MlmdListContext.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+interface MlmdOrderBy {
+  field: string;
+  direction: 'asc' | 'desc';
+}
+
+interface MlmdListContextProps {
+  filterQuery: string | undefined;
+  pageToken: string | undefined;
+  maxResultSize: number;
+  orderBy: MlmdOrderBy | undefined;
+  setFilterQuery: (filterQuery: string | undefined) => void;
+  setPageToken: (pageToken: string | undefined) => void;
+  setMaxResultSize: (maxResultSize: number) => void;
+  setOrderBy: (orderBy: MlmdOrderBy | undefined) => void;
+}
+
+const MlmdListContext = React.createContext({} as MlmdListContextProps);
+
+export const MlmdListContextProvider: React.FC<React.PropsWithChildren> = ({ children }) => {
```

Comment:
Why do we need a context provider for just the table?

Reply:
I added the context so that Juntao can use it for Executions as well. It would be for any list table using MLMD resources.

Reply:
closing as convo is in slack